js/components/plates.js

```diff
@@ -179,40 +176,41 @@ export default class Plates extends PureComponent {
     return config.authoring
   }
 
+  postMessageToModel (data) {
+    const { modelState } = this.state
+    // Most of the messages require model to exist. If it doesn't, queue messages and send them when it's ready.
+    if (modelState === 'loaded' || data.type === 'load' || data.type === 'unload') {
+      this.modelWorker.postMessage(data)
+    } else {
+      this.modelMessagesQueue.push(data)
+    }
+  }
+
+  postQueuedModelMessages () {
```

Comment:
This queueing looks good...it will be especially nice not to have to wait for loading to complete when we're just skipping through steps